plugins/kotlin/j2k/k1.new.post-processing/src/org/jetbrains/kotlin/idea/j2k/post/processing/processings/ConvertGettersAndSettersToPropertyProcessing.kt

```diff
@@ -570,6 +576,9 @@ private class ClassConverter(
         if (getterVisibility != null) {
             ktProperty.setVisibility(getterVisibility)
         }
+        if (realGetter?.function?.modifierList?.hasModifier(OVERRIDE_KEYWORD) == true) {
```

Comment:
This is the important change--it fixed a few K1 unit tests that broke when I commented out `diagnosticBasedProcessing(AddModifierFixFE10.createFactory(KtTokens.OVERRIDE_KEYWORD), Errors.VIRTUAL_MEMBER_HIDDEN)` (and probably some K2 tests, too)